### ISSUE TITLE
MongoDB compatibility issue

### ISSUE THREAD

**Issue by gelim** [pwndoc/pwndoc#194]
Hey,

it seems this commit 361cd0a8171bafc5cab3ab6db67b0d1f0f03832f is breaking install for people having 5.x being deployed (5.0.1 here).
I see the `setFeatureCompatibilityVersion` cannot be set to something lower than 4.4 

If you attempt something on the 5.0.1 db:

```
> db.adminCommand( { setFeatureCompatibilityVersion: "4.2" } )
{
        "ok" : 0,
        "errmsg" : "Invalid value for featureCompatibilityVersiondocument in admin.system.version, found 4.2, expected '4.4' or '4.9' or '5.0. See https://docs.mongodb.com/master/release-notes/4.4-compatibility/#feature-compatibility.",
        "code" : 4926900,
        "codeName" : "Location4926900"
}
> 
```

Is there any issues by fixing at least a 4.4 version in order to not block people using 5.x (not that I set the version by myself, it seems it has been pulled by default at install time)

Cheers

**Comment by kosztyua** [comment]
I had a similar issue after upgrading an ancient version of pwndoc to current. My fix was deliberately starting a mongo:4.4 and all seems to work fine. I understand 4.2.15 should have full compatibility but I am yet to encounter any issue with 4.4

**Comment by moscowchill** [comment]
Could you maybe explain how you fixed this issue as in steps?
